datafusion/src/physical_plan/sorts/sort_preserving_merge.rs

```diff
@@ -280,7 +287,11 @@ pub(crate) struct SortPreservingMergeStream {
     ///
```

Comment:
it looks like the comments here need to be updated (they still refer to `SortKeyCursor`)